third-party/rspamd/README.md

```diff
@@ -0,0 +1,40 @@
+## Additional webadmin endpoints
+
+### Report spam messages to RSpamD
+One can use this route to schedule a task that reports spam messages to RSpamD for its spam classify learning.
+
+```bash
+curl -XPOST 'http://ip:port/rspamd?action=reportSpam
+```
+
+This endpoint has the following param:
+- `action` (required): need to be `reportSpam`
+- `messagesPerSecond` (optional): Concurrent learns performed for RSpamD, default to 10
+- `period` (optional): duration (in seconds), only messages between `now` and `now - duration` are reported. By default, 
+all messages are reported.
```

Comment:
Please allow to provide unit. EG:

`curl -XPOST 'http://ip:port/rspamd?action=reportSpam&period=1d`

This makes it way more friendly